components/camel-opentracing/src/main/java/org/apache/camel/opentracing/GetBaggageProcessor.java

```diff
@@ -24,6 +24,7 @@
 import org.apache.camel.spi.IdAware;
```

Comment:
This will probably create conflicts with this #4001 once we merge it

Reply:
I see :( I'll fix those conflicts.